### DISCUSSION TITLE
Navigation 2020 Audi TT

### DISCUSSION THREAD

**Discussion by sprintfalcon** [Mr-MIBonk/M.I.B._More-Incredible-Bash#254] [General]
I went through all the steps to install MIB and update the fec.  I checked that all the codes were listed as "legal" in the Red menu.  I added the NAR maps and I still don't get navigation from the button or from the menu.  I re-ran MIB and checked that navigation was listed as installed.  I then checked the GPS antenna and it's listed as "open" - not installed?  Any further assistance is appreciated.

**Comment by harman-f** [comment]
no fec no maps ;)

Share LOG file

**Reply by sprintfalcon** [reply]
[MU1389-JTB-07416.11.1911760099.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11773525/MU1389-JTB-07416.11.1911760099.log)
Is this the correct one?

**Comment by cephalopodsquid** [comment]
I have the same issue with Nav on TT, but in EU region.  CarPlay works, I can see NavDB version, but no Nav option in Menu. 

Updated FW to MHI2_ER_AU43X_P5098, installed latest MIB, run AIO patch to activate CP/AA/etc and then also Nav activate separately.... so FEC definitely there and I can see 023 application license code is Legal in Engineering Menu.

See log attached.
[MU1339-JTH-07613.12.17340800B7.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11867759/MU1339-JTH-07613.12.17340800B7.log)

**Reply by cephalopodsquid** [reply]
Original EU maps (from 2017/18)
![IMG_6151](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/10574642/e6760484-c7a1-4be0-bdd5-1a3f94c72b67)

**Reply by harman-f** [reply]
try to run navigation activation again. In log it looks loike you did start multiple scripts in parallel.

**Reply by cephalopodsquid** [reply]
Thanks for the suggestion, but I after running Nav activation again - I still have no Navigation Menu or Button working.   Do the maps need to be newer with later versions of firmware perhaps?

**Reply by harman-f** [reply]
Have not seen this issue before.
Out of ideas for now...

Do you have maps installed on unit?

**Reply by sprintfalcon** [reply]
Yes. I did. However, the antenna search said open so I assume I don't have a GPS antenna.

**Reply by harman-f** [reply]
they still should show up.

**Comment by cephalopodsquid** [comment]
> Have not seen this issue before. Out of ideas for now...
> 
> Do you have maps installed on unit?

Yes as above screenshot 2017/18 EU maps....  do you think they might need updating?

**Comment by clown522** [comment]
Any new input, same problem here.
